src/python/pants/backend/python/typecheck/mypy/older_mypy_for_testing.lock

```diff
@@ -9,7 +9,7 @@
 //     "CPython<4,>=3.7"
 //   ],
 //   "generated_with_requirements": [
-//     "mypy==0.961"
+//     "mypy[python2]==0.961"
```

Comment:
The mypy tests that used this lock only accidentally worked before.  They were relying on python3.7 bringing along the same deps as this extra:

```
            "typed-ast<2,>=1.4.0; extra == \"python2\"",
            "typed-ast<2,>=1.4.0; python_version < \"3.8\"",

```